MiaWrapper.py

```diff
@@ -24,8 +24,18 @@ def miaWrapper(computationDataJson, paramsFile, outputPath):
     roiName = volumesOfInterest[0]['rois'][0]
    
     PyradiomicsWrapper.performRadiomicsComputation(ctDirectory, rtStructDirectory, roiName, outputPath, paramsFile)
-    
-    
+
+def checkIfCopyIsNeeded(images, modalities):
```

Comment:
I would prefer to check for all modalities. However, see below for a much simpler alternative solution.